environment_setup/setup_grsim_arch.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~#
+# UBC Thunderbots Arch Linux grSim Setup
+#
+# This script will install all the required libraries and dependencies to run
+# grSim. This only includes the simulator (grSim), and not anything related to
+# our AI. See the setup_software.sh script for that.
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~#
+
```

Comment:
Could we please put a big:
"WARNING: THIS IS NOT AN OFFICIALLY SUPPORTED SCRIPT. IT IS NOT RUN IN CI AND HAS NO GUARANTEE OF WORKING" warning here? Preferably in a big ASCII box.

Reply:
added